src/shouwToUi.js

```diff
@@ -0,0 +1,112 @@
+          li.appendChild(dots);
+          tasks.push(data);
+
+          dots.addEventListener('click', () => {
+            if (tasks.length > 0) {
+              tasks.map((task) => {
+                if (task.description === spanDec.innerHTML) {
+                  if (dots.innerHTML === '⋮') {
+                    dots.innerHTML = '&#x1F4BE;';
+                    spanDec.contentEditable = 'true';
+                    spanDec.addEventListener('input', () => {
+                      task.description = spanDec.textContent;
+                      localStorage.setItem('tasks', JSON.stringify(tasks));
+                    });
+                  } else {
+                    dots.innerHTML = '⋮';
+                    spanDec.contentEditable = 'false';
+                  }
+                }
+                return task;
+              });
+            }
+          });
+
+          checkBox.addEventListener('change', (e) => {
+            if (tasks.length > 0) {
+              tasks.map((task) => {
+                if (task.description === spanDec.innerHTML) {
+                  if (checkBox.checked) {
+                    task.completed = true;
+                    dots.innerHTML = '&#128465;';
+                    spanDec.classList.add('over-line');
+                    localStorage.setItem('tasks', JSON.stringify(tasks));
+                    dots.addEventListener('click', () => {
+                      dots.innerHTML = '&#128465;';
+                      if (task.completed === true) {
+                        const conIndex = tasks.indexOf(task);
+                        tasks.splice(conIndex, 1);
+                        e.target.parentNode.remove();
+                        localStorage.setItem('tasks', JSON.stringify(tasks));
+                      }
+                    });
+                  } else {
+                    task.completed = false;
+                    dots.innerHTML = '⋮';
+                    spanDec.classList.remove('over-line');
+                    localStorage.setItem('tasks', JSON.stringify(tasks));
+                  }
+                }
+                return task;
+              });
+            }
+          });
+          if (data.completed === true) {
+            spanDec.classList.add('over-line');
+            checkBox.checked = true;
+            dots.innerHTML = '&#128465;';
+            dots.addEventListener('click', (e) => {
+              tasks.map((task) => {
+                const conIndex = tasks.indexOf(task);
+                tasks.splice(conIndex, 1);
+                e.target.parentNode.remove();
+                localStorage.setItem('tasks', JSON.stringify(tasks));
+                return task;
+              });
+            });
+            ul.appendChild(li);
+          } else {
+            spanDec.classList.remove('over-line');
+            checkBox.checked = false;
+            ul.appendChild(li);
+          }
+          return data;
+        });
+      }
+    }
+  });
+}
```

Comment:
function printTasks() from the module getTask.js has a lot of similarities with this function. Both create the same HTML elements (li, the three dots, the checkboxes etc), both call events listeners on the same elements for doing similar tasks.
For using the dry principle, it would be better to refactor the code.